Cargo.toml

```diff
@@ -8,7 +8,11 @@ edition = "2018"
 lto = "fat"
 
 [features]
+default = ["alsasound", "pulseaudio"]
 damage_debug = []
+alsasound = ["alsa"]
```

Comment:
However, it is consistent with the naming of the libraries it depends on. If rename `alsasound`, I'd stick to `asound`. It is not possible to rename the feature to`alsa`, because the dependency is called `alsa` itself. We can rename it though, but I don't think it is a good idea.

Reply:
Well, for me the only that matters is the dependency. I avoid using pulseaudio and I don't wish to have it in my packages. It is possible to put the list of features and their dependencies in README.md, like
```md
* alsa-widget:  alsa-lib
* pulseaudio-widget: libpulse```

By the way, since pulseaudio depends on `libpulse`, it could be `pulse-audio` and `alsa-audio`.

Reply:
Feature flags control _features_, not _dependencies_. That they spare you of dependencies is an indirect (positive) side-effect.

"-audio" is technically fine, but the proper name is "pulseaudio", and "pulseaudio-audio" sounds silly. I think "alsa-widget" and "pulseaudio-widget" is fine.

Reply:
*For me*. I don't want my personal build to depend on pulseaudio, and it is far easier to check what it depends on if features are named after the dependencies they require.

But, after all, you're the author, so be it, I'll change it.

Reply:
Thank you.